packages/frontend/src/utils/ens.ts

```diff
@@ -22,3 +22,15 @@ export async function getEnsAvatar(ensNameOrAddress: string) {
     // noop
   }
 }
+
+export async function getEnsAddress(ensName: string) {
+  if (!ensName.endsWith('.eth')) {
```

Comment:
Technically there can be other ENS other than `.eth`, for example

![DeepinScreenshot_select-area_20220113175234](https://user-images.githubusercontent.com/168240/149437825-623e9d48-0c8e-424e-985d-a4df766b275c.png)

I think we should attempt to resolve any ENS name here